src/env.d.ts

```diff
@@ -9,6 +9,7 @@ interface ImportMeta extends Readonly<Record<string, unknown>> {
     PROD: boolean;
     DEV: boolean;
     SSR: boolean;
+    VITE_TEST_SUITE: string;
```

Comment:
Do I remember correctly that these can only be primitive types, e.g. number, boolean, string? Otherwise we could put the string literals here `"playwright"` and `"vitest"`.